### PR TITLE
REL-2804: ITC table tooltips

### PR DESCRIPTION
**Pull request by sraaphorst**
As per the JIRA task:

1. Added new and changed existing tooltips.
2. Made tooltips appear for table header.

The second task is actually nontrivial, and was done based on a Java example as provided here:
http://docs.oracle.com/javase/tutorial/uiswing/examples/components/TableToolTipsDemoProject/src/components/TableToolTipsDemo.java

### CODE REVIEW

File: bundle/jsky.app.ot/src/main/scala/jsky/app/ot/editor/seq/ItcTable.scala [hunk 1]
```diff
@@ -59,7 +58,6 @@ trait ItcTable extends Table {
   peer.setRowSelectionAllowed(true)
   peer.setColumnSelectionAllowed(false)
   peer.setSelectionMode(ListSelectionModel.SINGLE_SELECTION)
-  peer.getTableHeader.setReorderingAllowed(false)
```
Comment (cquiroz) [inline bundle/jsky.app.ot/src/main/scala/jsky/app/ot/editor/seq/ItcTable.scala]: Why do you disable reordering?

Comment (sraaphorst) [inline bundle/jsky.app.ot/src/main/scala/jsky/app/ot/editor/seq/ItcTable.scala]: There's no reason for it, I guess, apart from the fact that the default ordering makes sense.
I was just maintaining what was already in place.

Comment (cquiroz) [inline bundle/jsky.app.ot/src/main/scala/jsky/app/ot/editor/seq/ItcTable.scala]: Did you try if the tooltips are ok after you reorder? Sometimes the table indexes get out of whack when you reorder

Comment (sraaphorst) [inline bundle/jsky.app.ot/src/main/scala/jsky/app/ot/editor/seq/ItcTable.scala]: Yes, I initially had forgotten to turn reordering back on, and the tooltip positions were maintained as they should be. I believe that the call to `getModelIndex` handles this.

**Comment by cquiroz** [comment]
Looks good, I relaunched travis as it failed but I think it is unrelated

**Pull request merged**